common/app/model/meta/LinkedData.scala

```diff
@@ -29,8 +38,8 @@ object Logo {
 }
 
 case class Guardian(
-  override val `@type`: String = "Organization",
-  override val `@context`: String = "http://schema.org",
+  `@type`: String = "Organization",
+  `@context`: String = "http://schema.org",
   `@id`: String = "https://www.theguardian.com#publisher",
   name: String = "The Guardian",
   url: String = "http://www.theguardian.com/",
```

Comment:
`https`?

Reply:
hmm interesting. Yes that should be changed!